cmd/cluster-agent/app/clusterchecks.go

```diff
@@ -12,4 +12,5 @@ import "github.com/DataDog/datadog-agent/cmd/cluster-agent/commands"
 
 func init() {
 	ClusterAgentCmd.AddCommand(commands.GetClusterChecksCobraCmd(&flagNoColor, &confPath, loggerName))
+	ClusterAgentCmd.AddCommand(commands.RebalanceClusterChecksCobraCmd(&flagNoColor, &confPath, loggerName))
```

Comment:
shouldn't be a subcommand of the `clusterchecks` command?
`# agent clusterchecks rebalance`